packages/@aws-cdk/cloudformation-diff/lib/format.ts

```diff
@@ -44,25 +41,40 @@ export function formatDifferences(stream: NodeJS.WriteStream, templateDiff: Temp
  * Renders a diff of security changes to the given stream
  */
 export function formatSecurityChanges(stream: NodeJS.WriteStream, templateDiff: TemplateDiff, logicalToPathMap: {[logicalId: string]: string} = {}) {
-  const formatter = new Formatter(stream, logicalToPathMap);
-  formatter.readConstructPathsFrom(templateDiff);
+  const formatter = new Formatter(stream, logicalToPathMap, templateDiff);
 
+  formatSecurityChangesWithBanner(formatter, templateDiff);
+}
+
+function formatSecurityChangesWithBanner(formatter: Formatter, templateDiff: TemplateDiff) {
+  if (!templateDiff.iamChanges.hasChanges && !templateDiff.securityGroupChanges.hasChanges) { return; }
   formatter.formatIamChanges(templateDiff.iamChanges);
   formatter.formatSecurityGroupChanges(templateDiff.securityGroupChanges);
+
+  formatter.warning(`(This feature is experimental -- it might not be complete. See http://bit.ly/cdk-2EhF7Np)`);
```

Comment:
TBH, I am not even sure this is just because this is an experimental feature. There could always be security related changes at the application level that we will never know about. So maybe something like this:

```
WARNING: there may be security-related changes which do not appear in this list. See <URL> for details.
```

Maybe we should ask our security folks what would be the best wording here?